src/test/java/com/feelmycode/parabole/controller/OrderControllerTest.java

```diff
@@ -146,7 +149,7 @@ public void getOrderList() {
                     fieldWithPath("data.[].productPrice").type(JsonFieldType.NUMBER).description("상품 가격"),
                     fieldWithPath("data.[].productDiscountPrice").type(JsonFieldType.NUMBER).description("상품 할인 가격"),
                     fieldWithPath("data.[].productThumbnailImg").type(JsonFieldType.STRING).description("상품 썸네일 이미지"),
-                    fieldWithPath("data.[].updatedAt").type(JsonFieldType.STRING).description("주문 생성 일자")
+                    fieldWithPath("data.[].updatedAt").description("주문 생성 일자")
```

Comment:
일시는 어떤식으로 전송하는지 전송 형태를 작성해주세요

아래는 예시고 알맞게 수정해서 업데이트 해주세요

```suggestion
                    fieldWithPath("data.[].updatedAt").description("주문 생성 일자 (yyyy-MM-dd'T'HH:mm:ss)")
```

Reply:
설명 json방식으로 추가했어용
```
fieldWithPath("data.orderBySellerDtoList.[].orderInfoResponseDtos.[].state").type(JsonFieldType.STRING).description("상세 주문 상태. {\'BEFORE_PAY\': \'입금 전\', \'PAY_COMPLETE\': \'주문완료(결제)\', \'BEFORE_DELIVERY\': \'배송 준비\', \'DELIVERY\': \'배송 중\', \'DELIVERY_COMPLETE\': \'배송 완료\', \'BEFORE_ORDER\': \'주문 전\', \'ORDER_CANCEL\': \'주문 취소\', \'REFUND\': \'환불\', \'ERROR\': \'에러\'}"),
```